packages/graphql/src/services/learningElements.ts

```diff
@@ -96,6 +97,7 @@ function evaluateQuestionResponse(
             pointsPercentage,
             multiplier: multiplier ?? 1,
           }),
+          percentile: pointsPercentage ?? 0,
```

Comment:
should we add these defaults in the grading package or does it make more sense here?

Reply:
After checking the implementation again, I think it makes sense to leave it like this as 0 and null have different meanings in the grading package. Null is only returned, if no sample solutions are specified (i.e. content cannot be graded) and 0 is only returned when the answer was entirely wrong. Even though the case that there are no sample solutions cannot occur for learning elements (as we do not allow the use of such questions), it probably makes sense to still distinguish these cases in the grading package.